shims/spark310/src/main/scala/com/nvidia/spark/rapids/shims/spark310/GpuHashJoin.scala

```diff
@@ -47,6 +47,8 @@ object GpuHashJoin {
 
 trait GpuHashJoin extends GpuExec with HashJoin {
 
+  override def supportCodegen: Boolean = false
```

Comment:
lets add a comment here referring to https://github.com/NVIDIA/spark-rapids/issues/489